manifests/i/IObit/DriverBooster/10.3.0.124/IObit.DriverBooster.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0bugs
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster
+PackageVersion: 10.3.0.124
```

```suggestion
PackageVersion: 10.3.0
```
This should match control panel, or it will cause an upgrade loop. The version folder will need to be renamed